.github/workflows/ci.yml

```
@@ -44,3 +44,27 @@ jobs:
+  deploy-horarios:
+    needs: verificacao
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 24
+          cache: npm
+      - run: npm ci
+      - name: Schema e carga do D1 (idempotente: DELETE + INSERT versionados)
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)